src/styles/_catalog.scss

```diff
@@ -14,9 +14,17 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition-duration: 300ms;
+    transition-timing-function: linear;
```

Comment:
You should always add transition-property

Reply:
You're right. Thanks. I added.